doc/release/release_0.18.rst

```diff
@@ -1,3 +1,59 @@
+scikit-image 0.18.2
+===================
+
+We're happy to announce the release of scikit-image v0.18.2!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+This release mostly serves to add wheels for the aarch64 architecture, as well
+as fixing a couple of minor bugs.
```

Comment:
"(...) serves to add (...) as well as (to) fix (...)" although it doesn't really *serve* to fix, rather it *happens* to fix minor bugs as well... :thinking: 
Maybe: "This release mostly serves to add wheels for the aarch64 architecture; it also
fixes a couple of minor bugs."

Reply:
Can you make that a suggestion so I can click and commit? :smile:

Reply:
Sure! Approving with the suggested change.